10	5	match_alerts_app/nested_in_other_if/{old => new}/code.java

diff --git a/match_alerts_app/nested_in_other_if/old/code.java b/match_alerts_app/nested_in_other_if/new/code.java
index 311c341..d070f7b 100644
--- a/match_alerts_app/nested_in_other_if/old/code.java
+++ b/match_alerts_app/nested_in_other_if/new/code.java
@@ -21,5 +21,10 @@ class ClassX extends ClassY implements InterfX {
-        if (paramZ)
-            fieldX = paramW;
-        else{
-            fieldX = 0;
-        }
+		if(paramZ == 0){
+            if (paramZ)
+                fieldX = paramW;
+            else{
+                fieldX = 0;
+            }
+		}	
+	    else{
+		    fieldX = 1;
+		}
